AUTHORS

```diff
@@ -34,5 +34,4 @@ Jasper van Riet <jaspervanriet@gmail.com>
 Mattijs Fuijkschot <mattijs.fuijkschot@gmail.com>
 TruongSinh Tran-Nguyen <i@truongsinh.pro>
 Sander Dalby Larsen <srdlarsen@gmail.com>
-Marco Scannadinari <m@scannadinari.co.uk>
```

Comment:
uh, let's not take people out of the AUTHORS file...

Reply:
He was the person who added the super ellipse that I'm taking out... So I feel like it makes sense?

Reply:
Unfortunately that wasn't me - I wish I was that mathematically inclined!

https://github.com/flutter/flutter/pull/26295/files#diff-3d350169560e75d0cf9fc8e3574a3639

The diff is a bit misleading though

Reply:
Yah I just realized it's actually the other person, I'll leave both of you guys in tho, sorry about that tho 😟

Reply:
No worries :smiley: